src/database/connectDatabase.ts

```diff
@@ -0,0 +1,10 @@
+import {connect} from 'mongoose';
+
+export const connectDatabase = async () => {
+  await connect(process.env.MONGO_URI as string, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+    useFindAndModify: false,
+  });
+  console.log('Connected to database!');
```

Comment:
Out of curiosity is there a logger library for JS? Rather than simply just doing console.log

Reply:
I'll create an issue for it and look into it soon.